### ISSUE TITLE
writing single element value to 

### ISSUE THREAD

**Issue by dopplershift** [Unidata/netcdf4-python#117]
_From [space.8...@gmail.com](https://code.google.com/u/105207912851057280857/) on April 10, 2012 16:57:04_

Let us say I create a variable with a single element and write a python string to it.
Like so:
orthog_profile = Dataset('test.nc', 'w', 'NETCDF4_CLASSIC')
dum_dim = orthog_profile.createDimension('profile', 1)
cast_name = orthog_profile.createVariable('cast_name', str, 'profile')

data = np.array(['V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01',
 'V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01', 'V09B01'],
      dtype='|S6')

cast_name[:] = data[0]

I get the following error:

IndexError: single element VLEN slices must be specified by integers only

OK then if I create a variable with a dimension size of 2.

dum_dim_2 = orthog_profile.createDimension('profile_2', 2)
cast_name_2 = orthog_profile.createVariable('cast_name_2', str, 'profile_2')

cast_name_2[:] = data[0:2]

I get the same error

Basically I am having trouble writing multiple character strings.

I am using using version 0.9.9 with python version 2.6.6. on windows 7

Thank you in advance for your help.

Nathan

_Original issue: http://code.google.com/p/netcdf4-python/issues/detail?id=117_
